applications/launchpad/gui-react/src/commands/index.ts

```diff
@@ -1,22 +1,27 @@
 import { invoke, os } from '@tauri-apps/api'
-import { ChildProcess, Command } from '@tauri-apps/api/shell'
 
 /**
  * Check the Docker version on the host machine
- * @returns {Promise<ChildProcess>}
+ * @returns {Promise<string>}
  */
-export const dockerVersionCmd = async (): Promise<ChildProcess> => {
-  const command = new Command('docker', ['-v'])
-  return command.execute()
+export const checkDocker = async () => {
+  try {
+    const res: string = await invoke('check_docker')
```

Comment:
```suggestion
    const res: VersionInfo = await invoke('check_docker')
```

You also need to define the `VersionInfo` interface obviously: https://docs.rs/bollard/0.13.0/bollard/system/struct.Version.html